services/QuillLMS/client/app/bundles/Comprehension/components/studentView/container.tsx

```diff
@@ -92,12 +101,62 @@ export class StudentViewContainer extends React.Component<StudentViewContainerPr
     }
 
     window.addEventListener('keydown', this.handleKeyDown)
+    window.addEventListener('mousemove', this.resetTimers)
+    window.addEventListener('mousedown', this.resetTimers)
+    window.addEventListener('click', this.resetTimers)
+    window.addEventListener('keypress', this.resetTimers)
+    window.addEventListener('visibilitychange', this.setIdle)
```

Comment:
I super-love this approach of just attaching to standard event listeners for tracking and just writing a couple of functions.  Good call.